src/application.rs

```diff
@@ -69,66 +75,86 @@ impl Application {
 
         let (_, server_addr) = self.network.listen_tcp("0.0.0.0:0").unwrap();
         let server_port = server_addr.port();
-        
-        self.network.listen_udp_multicast(self.discovery_addr).unwrap();
+
+        self.network
+            .listen_udp_multicast(self.discovery_addr)
+            .unwrap();
 
         let discovery_endpoint = self.network.connect_udp(self.discovery_addr).unwrap();
-        self.network.send(discovery_endpoint, NetMessage::HelloLan(self.user_name.clone(), server_port)).unwrap();
+        self.network
+            .send(
+                discovery_endpoint,
+                NetMessage::HelloLan(self.user_name.clone(), server_port),
```

Comment:
I suppose that this style has been created by rust fmt. For example, in cases like this, I think that is much more readable to extract the message into a variable `let message = NetMesssge::HelloLan(...)` and let the send() method in one line. What do you think?

Reply:
Yes I agree

Personally I'm ok with any  style as long as it is automatically checked. 

If you have other suggestions I'll also apply them.